src/utils/helpers/list/mapColumnsByRole.ts

```diff
@@ -48,16 +50,19 @@ export const useMapColumnsByRole = () => {
       headerDetail: t('COLUMN_DEADLINE_DATE_TOOLTIP'),
       sortName: 'applicant-deadline',
       ColumnComponent: DeadlineCell,
+      hideOnMobileTest: (application) => !application.applicantDeadline,
```

Comment:
Can be a function so it can be conditional based on the current rowData. e.g. in this case the cell will be hidden if there is no deadline. (Also accepts a boolean to universally hide the cell)